### ISSUE TITLE
Invoke bot is not detecting the char selection screen

### ISSUE THREAD

**Issue by khaniri** [BigRedBot/NeverwinterInvokeBot#91]
As the subject says - it gets stuck at the char selection screen and wont progress further
Stuck at waiting for character selection screen. The computer and programs are responsive at the time - it just wont detect the character selection screen

**Comment by BigRedBot** [comment]
Could you give me a screen shot of the selection screen?

**Comment by khaniri** [comment]
![selec screen](https://cloud.githubusercontent.com/assets/23256282/20012147/f173f362-a2a5-11e6-85eb-092f15c4763b.jpg)

**Comment by BigRedBot** [comment]
Try using ImageCapture.exe and then try running ScreenDetection.exe and see what is says.

**Comment by khaniri** [comment]
screen detection detects the game client = 1024x768 window but it just wont pass the character selection screen to do the invoking

**Comment by cullukcan** [comment]
Same exact situation is happening to me , (also win10)

**Comment by AlanLeHun** [comment]
Yep. Somethings has happened/changed and I am now stuck at the same spot these past few days.

**Comment by BigRedBot** [comment]
ScreenDetection.exe should be saying:
"Character Selection Screen Detected"

If it is not, then it is not the script, it is just not detecting the image. At that point maybe set all the game's video and image settings to default to see if that is what is making it not see the image.

**Comment by AlanLeHun** [comment]
ScreenDetection does not get that far. It gives the screen dimensions and then nothing else. If I go into the game, it detects the Overflow Reward so the mechanics are still working.

For myself, this all started when I updated from 9.5 to 9.7 (pixel detection removed), however, an uninstall and reinstall of 9.5 still doesn't work.

![select1](https://cloud.githubusercontent.com/assets/12850439/20038301/0032b4fe-a429-11e6-88d0-7017dcb3c73c.png)

**Comment by BigRedBot** [comment]
Have you tried to resize the window? Drag the window to a different size and then let the script change the size back.

**Comment by AlanLeHun** [comment]
Resolved with a full re-install. What can I say, I'm impatient.

Prior to that I had replaced ScreenSelection which got me into the game and then had to replace OK to get past the Return to 'Character Selection' screen prompt but professions was still not working, so your worries that other images may have changed were well founded although clearly, not all of them had changed.

Started the re-install before your last post so sorry, I didn't try out the resize thing. That would have been so much easier had it worked.

I have no idea what could have caused this. I am happy with the way the game is now and did not change (or even look at) any settings prior to this failure.

Keep up the very good work, Red.
